.gitignore

```diff
@@ -20,4 +20,5 @@
 # Go workspace file
 go.work
 
-.env.secret
\ No newline at end of file
+.env.secret
```

Comment:
If we are excluding the `.env` file to keep all of the variable together, do we still need the `.env.secret` entry?

Reply:
Good point. Removed.